src/main/kotlin/ar/edu/unahur/obj2/servidorWeb/AnalizadorEstadistica.kt

```diff
@@ -0,0 +1,17 @@
+package ar.edu.unahur.obj2.servidorWeb
+
+import java.time.LocalDateTime
+
+class AnalizadorEstadistica() : Analizador() {
+    val respuestas = mutableListOf<Respuesta>()
+
+    override fun analizarRespuesta(respuesta: Respuesta) { this.respuestas.add(respuesta)}
+
+    fun promedioRespuestas() = respuestas.map { it.tiempo }.sum().div(respuestas.size)
```

Comment:
Está muy bien, también hay un método `average`:
```suggestion
    fun promedioRespuestas() = respuestas.map { it.tiempo }.average()
```